src/components/stat/stat.tsx

```diff
@@ -112,21 +123,32 @@ export const EuiStat: FunctionComponent<
     }
   );
 
+  const commonProps = {
+    'aria-hidden': true,
+  };
+
   const descriptionDisplay = (
     <EuiText size="s" className="euiStat__description">
-      <p aria-hidden="true">{description}</p>
+      {createElement(descriptionElement, commonProps, description)}
     </EuiText>
   );
 
+  const titlePropsWithColor = {
+    'aria-hidden': true,
+    style: {
+      color: `${titleColor}`,
+    },
+  };
+
+  const titleChildren = () => (isLoading ? '--' : title);
```

Comment:
No need for `titleChildren` to be a function,

```suggestion
  const titleChildren = isLoading ? '--' : title;
```

(and update the two places it is called)